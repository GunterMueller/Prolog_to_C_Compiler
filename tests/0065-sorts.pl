main :-
	sort([1,2,4,5,6,3,12,3,45,6,321,1], R), display(R), nl,
	merge([1,3,5,100], [2,a,b,e,h,x], R2), display(R2), nl,
	keysort([99-test, 1-a, 2-b, 3-xxx,0-e], R3), display(R3), nl.
