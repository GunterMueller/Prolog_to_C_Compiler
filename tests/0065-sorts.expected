[1,2,3,4,5,6,12,45,321]
[1,2,3,5,100,a,b,e,h,x]
[-(0, e),-(1, a),-(2, b),-(3, xxx),-(99, test)]
